encoded/PA1673_gradient/lowest_encoded.txt

```diff
@@ -0,0 +1,61 @@
+	0	1	2	3	4	5	6	7	8	9
+Glu_6Hour_(Pae_G1a).CEL	6.7398786544799805	0.0	0.0	5.70363187789917	0.0	6.582232475280762	0.0	0.0	0.0	0.0
+JS-A84.9.07.CEL	8.86330795288086	0.0	0.0	0.9941893219947815	0.13427108526229858	4.0595221519470215	0.0	0.0	0.0	0.0
```

Comment:
If you're saving these files with pandas, I'd do `float_format="%.5g"` to cut down on the excessive decimal places.